cfme/tests/ssui/test_ssui_myservice.py

```diff
@@ -56,7 +56,7 @@ def test_myservice_crud(appliance, setup_provider, context, order_catalog_item_i
         my_service.delete()
 
 
-@pytest.mark.uncollectif(lambda: current_version() < '5.8' or current_version() >= '5.9')
+@pytest.mark.uncollectif(lambda: current_version() < '5.7' or current_version() >= '5.9')
```

Comment:
We don't test for versions less that 5.7 . So no need for skipping less than 5.7.
@pytest.mark.uncollectif(current_version() >= '5.9')